meson_options.txt

```diff
@@ -0,0 +1,4 @@
+option('LIBUV_BUILD_TESTS', type: 'boolean', value: true, description: 'Build the unit tests when BUILD_TESTING is enabled and we are the root project')
+option('LIBUV_BUILD_BENCH', type: 'boolean', value: true, description: 'Build the benchmarks when building unit tests and we are the root project')
```

Comment:
Project options don't need to include the word "libuv" just because cmake -Doption sets a globally scoped variable in the CMakeLists.txt; also capitalizing the options is less common in meson.

Why does this mention "we are the root project"?

Reply:
In the original cmake, these are cmake_dependent_option with the requirement 
`CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR`

As you pointed out earlier, I think this is required because of some hardcoded path dependencies, which we solve later by setting the workdir of the test, so these are basically copy/pasted and not required anymore.